### DISCUSSION TITLE
xERROR in WSL

### DISCUSSION THREAD

**Discussion by afigueroa1xp** [JanDeDobbeleer/oh-my-posh#4259] [Troubleshoot] [answered]
I just personalized a theme and I'm getting the following error in WSL Ubuntu. When I use the default themes, I don't get the error:

![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/80425248/ab9ebcc7-d7f9-4c00-86de-6b5261523e4c)

That's not happen in PowerShell or Git Bash, only in WSL… ¿What could be happening?

**Comment by JanDeDobbeleer** [comment]
@afigueroa1973 this means there's a non-successful exit code somewhere (status segment).

**Comment by afigueroa1xp** [comment]
Starting from the fact that the theme was modified from an existing theme, I began to review each of the segments in the json file, finally I found the line that produces the error.

`
{
          "type": "status",
          "style": "plain",
          "foreground": "#b8ff75",
          "foreground_templates": [
            "{{ if gt .Code 0 }}#E06C75{{ end }}"
          ],
          "template": " x{{ reason .Code }}"
        },
`

I searched in the documentation, but couldn't find what the purpose of this segment is. I'm new to this, so some things I don't understand yet, any explanation would be welcome.

Thanks

**Reply by JanDeDobbeleer** [reply, marked as answer]
It displays the last know error code(s). Whenever you run a command, it has an exit code to let the executer know if it finished successful (0) or non-successful (non 0). When chaining command in a piper, certain shells also output multiple status'(PIPE_STATUS). In your case, on startup, something (unrelated to oh-my-posh) is having an issue and thus oh-my-posh displays that.

**Reply by afigueroa1xp** [reply]
Thanks
